src/libs/hooks/__tests__/useShowReview.test.tsx

```diff
@@ -6,6 +6,11 @@ import { useReviewInAppInformation } from 'features/bookOffer/services/useReview
 import { useShowReview } from 'libs/hooks/useShowReview'
 import { render } from 'tests/utils'
 
+const mockSettings = { disableStoreReview: false }
+jest.mock('features/auth/settings', () => ({
+  useAppSettings: jest.fn(() => ({ data: mockSettings })),
+}))
```

Comment:
pour changer la valeur des settings, je t'invite à t'inspirer de la manière dont s'est fait sur la recherche https://github.com/pass-culture/pass-culture-app-native/blob/master/src/features/search/components/__tests__/SearchResults.native.test.tsx

Reply:
Mais mockReturnValue doit être utilisé en dehors du it ?